java/Oleksii-Riabov/src/com/zoolatech/lecture1/tasks/_3/Task3.java

```diff
@@ -0,0 +1,26 @@
+package com.zoolatech.lecture1.tasks._3;
+
+import java.util.Scanner;
+
+/**
+ * Write a program that accepts a double number and outputs only its
+ * fractional part (all digits after the decimal point).
+ */
+
+public class Task3 {
+
+    public static void main(String[] args) {
+
```

Comment:
please pay attention to extra new lines: they might decrease the code readability

Reply:
Already remove redundant new lines after method start and before method end in all tasks.